src/main.rs

```diff
@@ -17,6 +17,7 @@ use std::error::Error;
 use std::fs;
 use std::io::{prelude::*, stdin};
 use std::path::PathBuf;
+use std::path::Path;
```

Comment:
You can write this as:
```
use std::path::{Path, PathBuf};
```
In fact, JetBrains RustRover has a function to optimise imports, which does this for you.